src/jbuf.c

```diff
@@ -750,11 +750,11 @@ int jbuf_stats(const struct jbuf *jb, struct jbuf_stat *jstat)
 int jbuf_debug(struct re_printf *pf, const struct jbuf *jb)
 {
 	int err = 0;
-	struct mbuf *mb = mbuf_alloc(512);
 
 	if (!jb)
 		return 0;
 
+	struct mbuf *mb = mbuf_alloc(512);
```

Comment:
We should null check `mb` here too.

Reply:
ah, your right. I'll make another PR.

Reply:
PR: https://github.com/baresip/baresip/pull/2814